R/rmarkdown.R

```diff
@@ -0,0 +1,41 @@
+
+#' Add an RMarkdown Template
+#'
+#' @param template_dir Name of the directory the template will live in within
```

Comment:
Could this default to `slug(template_name)`? (and hence be the second argument?)